### ISSUE TITLE
No Users Tab in TUI post-install

### ISSUE THREAD

**Issue by tailnet-h-usky-io** [JanssenProject/jans#7572]
**Describe the bug**
No `users` tab in TUI application post-install.

Hello, 

I am trying to install Janssen `v1.0.21` from [the helm charts][2]. _(Mostly because the charts for `v1.0.22` [reference a Docker tag][3] that does not (yet?) exist! When do Docker images get built...?)_

**To Reproduce**
1. Install Janssen via helm charts using MySQL-based persistence
2. Launch the TUI to create new users, [as documented][1]
3. Check top-level menu: `Auth Server`, `SCIM`, `Scripts`, `SMTP`, `Jans SAML`,  and `Jans TUI` – _it's missing `Users`!_

**Expected behavior**
A top-level menu including the `Users` tab, as depicted [in the docs][4].

**Actual Result**
A top-level menu with no `Users` tab. :( 
<img width="956" alt="image" src="https://github.com/JanssenProject/jans/assets/157428625/b4d29476-5775-4beb-bfdc-897ee425d1b4">

**Additional context**
Running on `microk8s` / `Ubuntu 22.04`, installed with [`startjanssendemo.sh`][4] _(no obvious install problems)_

Thought I'd check the DB, just to make sure it got initialized. 
```
mysql> select uid, mail, dn from jansPerson;
+-------+---------------------+------------------------------------------------------------+
| uid   | mail                | dn                                                         |
+-------+---------------------+------------------------------------------------------------+
| admin | admin@[my domain] | inum=76174769-e597-43af-8777-1223811d87d6,ou=people,o=jans |
+-------+---------------------+------------------------------------------------------------+
1 row in set (0.00 sec)
```
Pretty sure this account^^ was the one I used to make the initial TUI connection.

Any Idea why I can't see or add `Users`?

Thanks in advance!

[1]:https://docs.jans.io/v1.0.22/admin/usermgmt/usermgmt-cli-tui/#manage-data-in-jans-tui
[2]:https://github.com/JanssenProject/jans/tree/v1.0.21/charts/janssen#janssen
[3]:https://github.com/JanssenProject/jans/blob/e2e62108305c69e4ec38a809215819bc7911b184/charts/janssen/values.yaml#L50C12-L51C12
[4]:https://github.com/JanssenProject/jans/blob/v1.0.21/automation/startjanssendemo.sh

**Comment by tailnet-h-usky-io** [comment]
Quick update...

For what it's worth, the SCIM API seems to be working, at least for reading users:

``` 
❯ curl -k -G -H 'Authorization: Bearer xxxxxxxxxxxxxxxx' https://[MY DOMAIN]/jans-scim/restv1/v2/Users
{
    "schemas": [
        "urn:ietf:params:scim:api:messages:2.0:ListResponse"
    ],
    "totalResults": 1,
    "startIndex": 1,
    "itemsPerPage": 1,
    "Resources": [
        {
            "schemas": [
                "urn:ietf:params:scim:schemas:core:2.0:User"
            ],
            "id": "76174769-e597-43af-8777-1223811d87d6",
            "meta": {
                "resourceType": "User",
                "location": "https://[MY DOMAIN]/jans-scim/restv1/v2/Users/76174769-e597-43af-8777-1223811d87d6"
            },
            "userName": "admin",
            "name": {
                "familyName": "User",
                "givenName": "Admin",
                "middleName": "Admin",
                "formatted": "Admin Admin User"
            },
            "displayName": "Default Admin User",
            "nickName": "Admin",
            "active": true,
            "emails": [
                {
                    "value": "admin@[MY DOMAIN]",
                    "primary": false
                }
            ],
            "groups": [
                {
                    "value": "60B7",
                    "display": "Jannsen Manager Group",
                    "type": "direct",
                    "$ref": "https://[MY DOMAIN]/jans-scim/restv1/v2/Groups/60B7"
                }
            ]
        }
    ]
}
```
_I'll try writing a new one later today..._

**Comment by tailnet-h-usky-io** [comment]
Another update...

SCIM API works for both read and write operations.

Switched to LDAP backend. Still no `users` menu present in TUI, but SCIM API still works.

**Comment by ossdhaval** [comment]
Adding another observation:

I have 1.0.22 installed as a VM install. User tab is available in TUI.

**Comment by misba7** [comment]
Hello.
This happens due to the absence of env var `CN_CONFIG_API_PLUGINS="fido2,scim,user-mgt"`
Edit the the `config-api` deployment and the env
```
env:
   - name: CN_CONFIG_API_PLUGINS
      value: fido2,scim,user-mgt
```

And the users tab will load successfully.

**Comment by tailnet-h-usky-io** [comment]
Thanks @misba7 ! That did the trick.

**Comment by ossdhaval** [comment]
> Hello. This happens due to the absence of env var `CN_CONFIG_API_PLUGINS="fido2,scim,user-mgt"` Edit the the `config-api` deployment and add the env
> 
> ```
> env:
>    - name: CN_CONFIG_API_PLUGINS
>      value: fido2,scim,user-mgt
> ```
> 
> And the users tab will load successfully.

Hi @misba7 

Just to understand more, why is this env var not available by default in CN deployments? Is there a reason behind it?

**Comment by misba7** [comment]
> > Hello. This happens due to the absence of env var `CN_CONFIG_API_PLUGINS="fido2,scim,user-mgt"` Edit the the `config-api` deployment and add the env
> > ```
> > env:
> >    - name: CN_CONFIG_API_PLUGINS
> >      value: fido2,scim,user-mgt
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > And the users tab will load successfully.
> 
> Hi @misba7
> 
> Just to understand more, why is this env var not available by default in CN deployments? Is there a reason behind it?

Hi @ossdhaval 
I found it available by default in `1.0.22` (latest). We can troubleshoot why this env was specifically missing in `1.0.21`.

**Comment by ossdhaval** [comment]
@misba7 Thank you for your quick response. 

Great if it is available by default in 1.0.22. I don't think we need to troubleshoot 1.0.21 as it can be quickly worked around. :+1:
